controllers/provider-vsphere/example/old/00-componentconfig.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Why are all these example files in an `old` subdirectory?

Reply:
removed old directory